student.rb

```diff
@@ -1,6 +1,6 @@
-# require_relative './person'
+require_relative './person'
 class Student < Person
-  def initialize(classroom)
+  def initialize(classroom, name, age)
     super(name, age)
     @classroom = classroom
```

Comment:
- [ ] Getters for the instance variable `@classroom` should be present in the Student class.